src/test/java/com/google/spanner/liquibase/LiquibaseTests.java

```diff
@@ -903,7 +935,7 @@ static List<Map<String, Object>> testQuery(java.sql.Connection conn, String quer
 
     // Set readonly
     boolean readOnlyStatus = conn.isReadOnly();
-    conn.setReadOnly(true);
+    // conn.setReadOnly(true);
```

Comment:
Is this temporarily necessary? If so, can we add a TODO? Or otherwise revert the change?